### ISSUE TITLE
Landscape loading/saving broken

### ISSUE THREAD

**Issue by AaronVanGeffen** [OpenLoco/OpenLoco#765]
**Version information:**
 - Platform: Arch Linux
 - Version: 20.10+
 - Build/commit: 0ee80c5d2ab7b29ef0e7149efe0df3a916ace118

**Describe the bug**
Loading and saving of landscapes through the scenario editor appears to be broken. The landscape is saved as a .SC5 in the landscapes folder, but the resulting file is indicated to be corrupt when loading. It is roughly 700kB in size -- examples attached.

**Release unaffected**
This does not appear to happen with the v20.10 release, so this looks to be a regression.

**Example files**
[Unnamed.SC5.gz](https://github.com/OpenLoco/OpenLoco/files/5991652/Unnamed.SC5.gz)
[Unnamed2.SC5.gz](https://github.com/OpenLoco/OpenLoco/files/5991653/Unnamed2.SC5.gz)

**Comment by AaronVanGeffen** [comment]
I think this might be solved by #697. However, I currently can't test this on Linux.

**Comment by AaronVanGeffen** [comment]
```
aaron@izumo ~/Repositories/OpenLoco/OpenLoco ((c652fa97...)|BISECTING) $ git bisect log 
git bisect start
# bad: [e769ab749aa2f8d4943784c503aed7bbc3ea1ddb] Merge pull request #779 from AaronVanGeffen/fix/town-list
git bisect bad e769ab749aa2f8d4943784c503aed7bbc3ea1ddb
# good: [ee516cf78a2454c09e6b97416c1567b34efa4357] Update OpenLoco version to v20.10
git bisect good ee516cf78a2454c09e6b97416c1567b34efa4357
# bad: [1e8e3ae973258436e2ca852bc2ce44f77c37f65b] Move vehicle to vehicle.cpp and rename VehicleCommon
git bisect bad 1e8e3ae973258436e2ca852bc2ce44f77c37f65b
# good: [e7a0eae66e3fb7b142d3ddcba874ee2ce30529f4] Remove Travis CI (#704)
git bisect good e7a0eae66e3fb7b142d3ddcba874ee2ce30529f4
# bad: [9fd780a1f50d4cc7cbf8c20f09a9dc9736bfbeb0] Implement viewport render (#714)
git bisect bad 9fd780a1f50d4cc7cbf8c20f09a9dc9736bfbeb0
# good: [94b1b3d92434374737e8634a80ead2cd25b5bb0c] Use upper case for SV5 extension
git bisect good 94b1b3d92434374737e8634a80ead2cd25b5bb0c
# bad: [1646f637c9bb91f524991bdd83f9a984291f9792] Merge pull request #690 from IntelOrca/feature/autosave
git bisect bad 1646f637c9bb91f524991bdd83f9a984291f9792
# bad: [145774be182ac0a70ecb2c00a7a17a46bdedb586] Auto create save directory and reduce warnings
git bisect bad 145774be182ac0a70ecb2c00a7a17a46bdedb586
# bad: [c652fa97c295ba314afa3dec8f50b051f532d02f] Use constants for file extensions
git bisect bad c652fa97c295ba314afa3dec8f50b051f532d02f
# first bad commit: [c652fa97c295ba314afa3dec8f50b051f532d02f] Use constants for file extensions
```
